tests/drivers/gpio/gpio_basic_api/testcase.yaml

```diff
@@ -142,3 +142,9 @@ tests:
     extra_args: "DTC_OVERLAY_FILE=boards/frdm_rw612_hsgpio.overlay"
     harness_config:
       fixture: hsgpio_loopback
+  drivers.gpio.siwx91x_device_runtime_pm:
```

Comment:
I think this test is not specific to siwx91x. You could name it `drivers.gpio.pm`.